ersilia/core/session.py

```diff
@@ -42,12 +42,26 @@ def register_service_class(self, service_class):
         with open(self.session_file, "w") as f:
             json.dump(data, f, indent=4)
 
-    def open(self, model_id):
+    def tracking_status(self):
+        data = self.get()
+        if data is None:
+            return None
+        else:
+            return data["track_status"]
+
+    def register_tracking_status(self, track_status):
```

Comment:
Do we really need this method if we are serializing `track_status` when we first write to the session JSON? I don't think we need this method.

Reply:
You are right, i have removed it